messages/sv.json

```
@@ -2388,6 +2388,7 @@
     "createSelected": "Förhandsgranska krav i import ({count})",
     "createConfirm": "{count, plural, one {Skapa # krav i kravområde {area}?} other {Skapa # krav i kravområde {area}?}}",
     "createError": "Kunde inte skapa krav",
+    "generationFailed": "Genereringen misslyckades",
```

Biome (2.5.1)

[error] The key generationFailed was already declared.

(lint/suspicious/noDuplicateObjectKeys)